EMOBON00092/manual_example_ro-crate-metadata.json

```diff
@@ -54,174 +41,216 @@
                         {"@id": "sequence-categorisation"},
-                        {"@id": "taxonomy-summary"}
+                        {"@id": "taxonomy-summary"},
+                        {"@id": "EMOBON00092/provenance.json"}
                     ] 
     },
     {
-        "@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/DBH.merged.cmsearch.all.tblout.deoverlapped.dvc",
+        "@id": "EMOBON00092/provenance.json",
+        "@type": ["File", "CPMProvenanceFile"],
+        "conformsTo": {"@id": "https://by-covid.github.io/cpm-ro-crate/0.2/"},
+        "encodingFormat": "application/ld+json",
+        "name": "EMOBON00092 Provenanace CPM File",
+        "description": "CPM compliant provenance file for EMOBON00092",
+        "about": [
+            {"@id": "_:EMOBON00092-CPM-provgen"}
+        ]
+    },
+    {
+        "@id": "EMOBON00092/DBH.merged.cmsearch.all.tblout.deoverlapped",
         "@type": "File",
         "name": "Sequences hits list",
         "contentSize": 7751834,
         "encodingFormat": "text/plain",
-        "url": "https://emobon-lfs-demo.s3.mesocentre.uca.fr/files/md5/22/053538759fa3b405533e20c0cf6322",
-        "checksum" : "22053538759fa3b405533e20c0cf6322",
+        "downloadUrl": "https://emobon-lfs-demo.s3.mesocentre.uca.fr/files/md5/22/053538759fa3b405533e20c0cf6322",
+        "checksum" : {
+            "checksumValue":"22053538759fa3b405533e20c0cf6322"
+        },
         "description": "Sequence hits against covariance model databases. Mandatory to run partially the functional annotation step of metaGOflow"
     },
     {
-        "@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/DBH.merged.fasta.dvc",
+        "@id": "EMOBON00092/DBH.merged.fasta",
         "@type": "File",
         "name": "Merged filtered reads",
         "contentSize": 7477104308,
-        "encodingFormat": "text/plain",
-        "url": "https://emobon-lfs-demo.s3.mesocentre.uca.fr/files/md5/85/1bd7b6823e28f20d552f99fc3ce2cb",
-        "checksum" : "851bd7b6823e28f20d552f99fc3ce2cb",
+        "encodingFormat": "text/x-fasta",
+        "downloadUrl": "https://emobon-lfs-demo.s3.mesocentre.uca.fr/files/md5/85/1bd7b6823e28f20d552f99fc3ce2cb",
+        "checksum" : {
+            "checksumValue":"851bd7b6823e28f20d552f99fc3ce2cb"
+        },
         "description": "Merged filtered reads"
     },
     {
-        "@id": "https://github.com/emo-bon/mgf-s3-prototype/blob/main/EMOBON00092/DBH.merged.motus.tsv.dvc",
+        "@id": "EMOBON00092/DBH.merged.motus.tsv",
         "@type": "File",
         "name": "motus list",
         "contentSize": 1091,
-        "encodingFormat": "text/tab-separated-values",
-        "url": "https://emobon-lfs-demo.s3.mesocentre.uca.fr/files/md5/86/da0a637a532316a03536bce851764e",
-        "checksum" : "86da0a637a532316a03536bce851764e",
+        "encodingFormat": "https://www.iana.org/assignments/media-types/text/tab-separated-values",

```

Comment:
checked up in the ro-crate specification at https://www.researchobject.org/ro-crate/specification/1.2-DRAFT/data-entities.html#file-format-profiles

seems to be suggesting the encodingFormat could be an array that holds both the mime-type in string format and a nested `@id: to point to actual uri`

Reply:
also - would suggest to have a conversation started on not only capturing the (very generic) mime-type but also more details on the actual purpose / content-contract these kind of files represent

e.g. in this case of "motus list" it feels like there is more detail to share / explain about what these files contain, and how that could be parsed?

this should be added via conformsTo markers (uri)  expressing somewhat of a contract / agreed expectations / automated parsing support / profile of these

if these extra "content-description-uri' are broadly bio-genomics-oriented these should be defined on a level matching that scope (e.g. ebi, tdwg, ...) - if they are tied to emobon / our MetaGoFlow only, then we could list such terms ourselves 

maybe best to start conversation on this in a separate issue?  maybe within https://github.com/emo-bon/governance-data